functions/taskrouterListeners/transfersListener.private.ts

```diff
@@ -41,16 +48,29 @@ const isChatTransfer = (
 const isChatTransferAccepted = (
   eventType: EventType,
   taskChannelUniqueName: string,
-  taskAttributes: { transferMeta?: TransferMeta },
-) => eventType === RESERVATION_ACCEPTED && isChatTransfer(taskChannelUniqueName, taskAttributes);
+  taskAttributes: ChatTransferTaskAttributes,
+) =>
+  eventType === RESERVATION_ACCEPTED &&
+  isChatTransfer(taskChannelUniqueName, taskAttributes) &&
+  taskAttributes.transferTargetType === 'worker';
```

Comment:
Maybe renaming these functions to `isChatTransferToWorkerAccepted` and `isChatTransferToWorkerRejected`?